Topics/User_Experience/AB_Testing.md

```diff
@@ -0,0 +1,36 @@
+# A/B Testing
+
+## **Introduction:**
+A/B testing is a research method used to compare two versions of a web page or mobile app to determine which variation of the UI results in better performance. It allows individuals and companies to optimize user experience and make improvements to their application. 
+
+## **Steps/How To:**
+**Identify Changes:** Identify any potential problem areas within the UI and create a hypothesis of how to fix it <br />
+
+**Create Variation:** Create two versions of the applications UI to run the experiment on, create the new design based on the hypothesis, whether it be changing the font, colors, or re-designing the whole page. <br />
+- The original page is known as the control.
+- The altered page is known as the variation. 
+
+**Running The Experiment:** Randomly filter the traffic of the application where one group (Group A) sees the original, or controlled version, and the other group (Group B) sees the modified version. <br />
+
+**Collect Data:** Record and measure different metrics such as user session duration or other performance metrics related to the application. <br />
+
+**Analyze Results:** Based on the results, determine which version improves performance and deploy the winning variation. The experiment can be repeated to make further improvements. <br />
+
+## **Types of A/B Testing:**
```

Comment:
Are there pros/cons for each method? Could some methods be more “expensive” than others?

Reply:
I will add some cons for each strategy for readers to consider.